### PR TITLE
Update fips.adoc

### PR DESCRIPTION
**Pull request by stephenbjm**
Fixed some typographical and language usage issues.

### CODE REVIEW

**Comment by mposolda** [comment]
@stephenbjm Thanks for the PR!

@andymunro Could you please review this PR?

File: docs/guides/server/fips.adoc [hunk 1]
```diff
@@ -31,10 +30,10 @@ fips-mode-setup --enable
 
 == BouncyCastle library
 
-Keycloak internally uses the BouncyCastle library for many cryptography utilities. However, the default flavor of the BouncyCastle library that shipped with Keycloak is not FIPS compliant, 
-but, BouncyCastle also provides a FIPS validated version of it's library. The FIPS validated BouncyCastle library cannot be shipped with Keycloak due to license constraints and
+Keycloak internally uses the BouncyCastle library for many cryptography utilities. Please note that the default version of the BouncyCastle library that shipped with Keycloak is not FIPS compliant; 
+however BouncyCastle also provides a FIPS validated version of its library. The FIPS validated BouncyCastle library cannot be shipped with Keycloak due to license constraints and
```
Comment (andymunro) [inline docs/guides/server/fips.adoc:34]:
```suggestion
however,  BouncyCastle also provides a FIPS validated version of its library. The FIPS validated BouncyCastle library cannot be shipped with Keycloak due to license constraints and
```

File: docs/guides/server/fips.adoc [hunk 2]
```diff
@@ -53,7 +52,7 @@ You can create either `pkcs12` or `bcfks` keystore to be used for the Keycloak s
 
 The `p12` (or `pkcs12`) keystore (and/or truststore) works well in BCFIPS non-approved mode.
 
-PKCS12 keystore can be generated with OpenJDK 17 Java on RHEL 9 in the standard way. For instance this command can be used to generate such keystore:
+PKCS12 keystore can be generated with OpenJDK 17 Java on RHEL 9 in the standard way. For instance the following command can be used to generate the keystore:
```
Comment (andymunro) [inline docs/guides/server/fips.adoc:55]:
```suggestion
PKCS12 keystore can be generated with OpenJDK 17 Java on RHEL 9 in the standard way. For instance,  the following command can be used to generate the keystore:
```

File: docs/guides/server/fips.adoc [hunk 3]
```diff
@@ -63,20 +62,20 @@ keytool -genkeypair -sigalg SHA512withRSA -keyalg RSA -storepass passwordpasswor
   -dname CN=localhost -keypass passwordpassword
 ----
 
-When system is in FIPS mode, the default `java.security` file is changed in order to use FIPS enabled security providers, so no additional configuration is needed. Additionally, in PKCS12 keystore you can store PBE (password-based encryption) keys simply via the keytool command, which makes it ideal for using it with Keycloak KeyStore Vault and/or to store configuration properties in the KeyStore Config Source. For more details, see the <@links.server id="configuration"/> and the <@links.server id="vault"/>.
+When the system is in FIPS mode, the default `java.security` file is changed in order to use FIPS enabled security providers, so no additional configuration is needed. Additionally, in the PKCS12 keystore you can store PBE (password-based encryption) keys simply via the keytool command, which makes it ideal for using it with Keycloak KeyStore Vault and/or to store configuration properties in the KeyStore Config Source. For more details, see the <@links.server id="configuration"/> and the <@links.server id="vault"/>.
```
Comment (andymunro) [inline docs/guides/server/fips.adoc:65]:
```suggestion
When the system is in FIPS mode, the default `java.security` file is changed in order to use FIPS enabled security providers, so no additional configuration is needed. Additionally, in the PKCS12 keystore, you can store PBE (password-based encryption) keys simply by using the keytool command, which makes it ideal for using it with Keycloak KeyStore Vault and/or to store configuration properties in the KeyStore Config Source. For more details, see the <@links.server id="configuration"/> and the <@links.server id="vault"/>.
```

**Review by andymunro — CHANGES_REQUESTED** [review]
@mposolda a few changes to suggest.

**Review by andymunro — APPROVED** [review]

**Comment by mposolda** [comment]
@stephenbjm @andymunro Thanks for the review and updates!

**Review by mposolda — APPROVED** [review]

**Pull request merged**